pkg/private/tar/tar.bzl

```diff
@@ -225,6 +225,9 @@ def _pkg_tar_impl(ctx):
             files = depset([output_file]),
             runfiles = ctx.runfiles(files = outputs),
         ),
+        OutputGroupInfo(
```

Comment:
As you suggested, can you leave a comment that the specification is subject to change in any minor release.

Reply:
Friendly ping.

Reply:
Thanks for the ping, done.